plugins/nodes/src/js/components/NodesGridView.js

```diff
@@ -49,7 +49,7 @@ var NodesGridView = React.createClass({
     }
 
     var loadingClassSet = classNames({
-      'hidden': hasLoadingError
+      hidden: hasLoadingError
```

Comment:
The boyscout in me did this while I was editing this for testing purposes... I'll leave this unless somebody objects!